index.cjs

```diff
@@ -0,0 +1,104 @@
+const turoConfig = require("@open-turo/eslint-config-typescript");
+const jsxA11yPlugin = require("eslint-plugin-jsx-a11y");
+const reactPlugin = require("eslint-plugin-react");
+const reactCompilerPlugin = require("eslint-plugin-react-compiler");
+const reactHooksPlugin = require("eslint-plugin-react-hooks");
+const globals = require("globals");
+// TODO We can't do this because this function is not here directly technically
```

Comment:
I didn't understand this explanation, maybe we could extend it a bit? 😄

Reply:
I'll fix it. What I meant is that `typescript-eslint` is not a dependency in this repository, but a transitive dependency coming from `@open-turo/eslint-config-typescript`